backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/window/api/impl/ADWindowDAO.java

```diff
@@ -909,6 +912,20 @@ private void copyTabs(final I_AD_Window targetWindow, final I_AD_Window sourceWi
 			final I_AD_Tab targetTab = copyTab_SkipUISections(copyCtx, targetWindow, existingTargetTab, sourceTab);
 
 			sourceAndTargetTabs.add(ImmutablePair.of(sourceTab, targetTab));
+
```

Comment:
I think inead of this we need some override-columns, like we have them in (AFAIR) `AD_Element`.
IMHO that should be done in a follow-up-issue, because the current issue's bud is too small enayways, and it's the end of the sprint.
@teosarca wdyt?

Reply:
confirmed with Torby, will be done in anothere issue

Reply:
@metas-ts / @riadmen   let's talk about that on follow-up. ping me there.